File: PooCC01CNetz/Semana2/Python/Ejercicios4/MetodoObjeto.py
from Celular import *
class MetodoObjeto:
    def main(self):
        c = Celular()
        c.modelo = "jaguei"
        c.activo = True
        mo.recibeCelular(c)

    def recibeCelular(self, c):
        print("Modelo   : " + str(c.modelo))
        print("Precio   : " + str(c.precio))
        print("Activado : " + str(c.activo))

if __name__ == "__main__":
    mo = MetodoObjeto()
    mo.main()

File: PooCC01CNetz/Semana2/Java/Ejercicios4/src/MetodoObjeto.java
public class MetodoObjeto {
    public static void main(String[] args) {
        MetodoObjeto mo = new MetodoObjeto();
        Celular c = new Celular();
        c.modelo = ("jaguei");
        c.activo = true;
        mo.recibeCelular(c);
    }

    public void recibeCelular(Celular c) {
        System.out.println("Modelo   : " + c.modelo);
        System.out.println("Precio   : " + c.precio);
        System.out.println("Activado : " + c.activo);
    }
}
